packages/grid/_modules_/grid/hooks/features/treeData/useGridTreeData.ts

```diff
@@ -36,18 +38,18 @@ export const useGridTreeData = (
       headerName: apiRef.current.getLocaleText('treeDataGroupingHeaderName'),
       ...GRID_TREE_DATA_GROUP_COL_DEF_FORCED_FIELDS,
     };
-    const colDefOverride: Partial<GridColDef> = propGroupingColDef ?? {};
-    // let colDefOverride: Partial<GridColDef>;
-    //
-    // if (typeof propGroupingColDef === 'function') {
-    //   const params: GridColDefOverrideParams = {
-    //     colDef: baseColDef,
-    //   };
-    //
-    //   colDefOverride = propGroupingColDef(params);
-    // } else {
-    //   colDefOverride = propGroupingColDef ?? {};
-    // }
+    let colDefOverride: Partial<GridColDef>;
+
+    if (typeof propGroupingColDef === 'function') {
+      const params: GridGroupingColDefOverrideParams = {
+        treeGroupingName: TREE_DATA_GROUPING_NAME,
```

Comment:
```suggestion
        groupingName: TREE_DATA_GROUPING_NAME,
```

It could help to distinguish "grouping" features from "tree data"

Reply:
Done :+1: